### ISSUE TITLE
replace e2e script

### ISSUE THREAD

**Issue by FabienDehopre** [briebug/cypress-schematic#3]
During the installation of cypress, you add two script (`cypress-open` and `cypress-run`). You can also replace the `e2e` script by `npm run cypress-open` (instead of `ng e2e`).

**Comment by anthonymjones** [comment]
Closing this issue... I just released v3.0.0, which includes a builder to run `ng e2e` when protractor is removed. Thanks @FabienDehopre!

**Comment by jayoungers** [comment]
In case anyone stumbles upon this, I've updated my config like so for CI:

```json
        "e2e": {
          "builder": "@briebug/cypress-schematic:cypress",
          "options": {
            "devServerTarget": "my-app:serve",
            "watch": true,
            "headless": false
          },
          "configurations": {
            "production": {
              "devServerTarget": "my-app:serve:production",
              "watch": false,
              "headless": true
            }
          }
        }
```

Where it can be used like so:

`ng e2e --prod`
